spec/scrabble_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+require 'scrabble'
+
+describe Scrabble::Scrabble do
+
+  describe ".Scrabble" do
+    it "Has a score class method" do
+      expect(Scrabble::Scrabble).to respond_to :score
```

Comment:
Since you use the `Scrabble::Scrabble` object in every test, you might consider using the `let` helper method: https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let

This allows you to say something like `let(:scrabble) { Scrabble::Scrabble }` inside your `describe` block, and then each `it` block can simply use the variable `scrabble`. E.g. `expect(scrabble.score("dad")).to eq 5`